dockerfile/build.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gradle:7.2-jdk11-alpine AS build
+WORKDIR /home/gradle/src
+RUN git clone -b task_4 https://github.com/kkarnauk/se-project-fall-2021.git .
```

Comment:
Wow, why do you clone your repository here?

Reply:
fix